### ISSUE TITLE
Definition of the path to ./blender in config .xml not taken into account (for dynamic .blend mod at blenderVR start)

### ISSUE THREAD

**Issue by PyrApple** [BlenderVR/blender-vr#19]
While the configuration file (.xml) can be used to define the path towards blender binary*
it doesn't seem to be taken into account, as e.g. a misspelled  declaration throws no errors in BlenderVR run.

Line concerned in .xml configuration file:
 <starter blender='/blenderVR_git/blender-sources/build_darwin/bin/blender.app/Contents/MacOS/blender'>
[...]
  </starter>

*(blender bpy used to modify the .blend on the go to add blender-vr related features prior to the BlenderVR run)

**Comment by dfelinto** [comment]
In fact I think blender should be defined in the system, just like blenderplayer is.
It probably makes this fix simpler too.

**Comment by PyrApple** [comment]
Fine by me, 

(some may complain that you can't have blenderplayer nor blender pointing towards non blenderVR versions.)

David

**Comment by dfelinto** [comment]
By system I meant:
```
<computers>
  <system>
    <blender executable="MYSPECIALPATH/blender">
    <blenderplayer executable="MYSPECIALPATH/blenderplayer">
  </system>
(...)
</computers>
```

**Comment by PyrApple** [comment]
++
(no ambiguities that way)

**Comment by dfelinto** [comment]
For the records, I believe the reasoning behind this is that the loader is called only once, while the player is set for each of the clients.

**Comment by dfelinto** [comment]
Issue fixed (712bf70b) without changing the XML definitions
